tools/travis/build.sh

```diff
@@ -38,6 +38,11 @@ cat $WHISKDIR/whisk.properties
 cd $ROOTDIR/packages
 ./installCatalog.sh $WHISKDIR/config/keys/auth.whisk.system
 
+# Set credentials
+VCAP_SERVICES_FILE="$(readlink -f $ROOTDIR/tests/credentials.json)"
```

Comment:
The reference I mean put the template json under test folder, the template file is from that repo, also the script that modify the property file was referenced the repo, you could see the build file and the hierarchy from it.